tests/integration/targets/azure_rm_webapp/tasks/main.yml

```
@@ -16,26 +16,22 @@
     win_plan_name: "{{ (resource_prefix | replace('-', 'x'))[-8:] }}winplan"
     linux_plan_name: "{{ (resource_group_secondary | replace('-', 'x'))[-8:] }}linplan"
     slot1_name: "stage1"
-    location: "{{ __rg_info.resourcegroups.0.location }}"
 
-- name: Create User Managed Identities
-  azure_rm_resource:
-    resource_group: "{{ resource_group }}"
-    provider: ManagedIdentity
-    resource_type: userAssignedIdentities
-    resource_name: "{{ item }}"
-    api_version: "2023-01-31"
-    body:
-      location: "{{ location }}"
-    state: present
-  loop:
-    - "ansible-test-webapp-identity"
-    - "ansible-test-webapp-identity-2"
-
-- name: Set identities IDs to test. Identities ansible-test-webapp-identity and ansible-test-webapp-identity-2 have to be created previously
+- name: Set location and managed_identity_ids
   ansible.builtin.set_fact:
-    user_identity_1: "/subscriptions/{{ azure_subscription_id }}/resourcegroups/{{ resource_group }}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ansible-test-webapp-identity"
-    user_identity_2: "/subscriptions/{{ azure_subscription_id }}/resourcegroups/{{ resource_group }}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ansible-test-webapp-identity-2"
+    location: "{{ __rg_info.resourcegroups.0.location }}"
```

Because the quota of eastus has been full before, eastus cannot be created. The web application and project should be in the same area (resource_group_third), thank you!
```suggestion
    location: westus2
```